components/prombench/manifests/benchmark/1c_cluster-role-binding.yaml

```diff
@@ -1,11 +1,23 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: loadgen-scaler
+  namespace: prombench-{{ .PR_NUMBER }}
+rules:
+- apiGroups: ["apps"]
+  resources:
+  - deployments
+  verbs: ["get", "list", "update"]
+---
```

Comment:
did you try my idea to have a global role with full permissions instead of maintaining different roles for the different pieces?

Reply:
We can use the [Prometheus cluster-role](https://github.com/prometheus/prombench/blob/master/components/prombench/manifests/results/authenticate_rbac.yaml#L8) for loadgen-scaler (after add list/update actions to it).
But often granting an object more permission than it needs is not a good idea.